src/posit/connect/jobs.py

```diff
@@ -120,40 +122,42 @@ def destroy(self) -> None:
         ----
         This action requires administrator, owner, or collaborator privileges.
         """
-        self._ctx.session.delete(self._endpoint)
+        endpoint = self._ctx.url + self._path
+        self._ctx.session.delete(endpoint)
 
 
-class Jobs(
-    ActiveFinderMethods[Job],
-    ActiveSequence[Job],
-):
-    def __init__(self, ctx, parent: Active, uid="key"):
+class Jobs(ActiveFinderMethods[Job], ActiveSequence[Job]):
+    def __init__(self, ctx: Context, path: str, pathinfo: str = "jobs", uid: str = "key"):
         """A collection of jobs.
 
         Parameters
         ----------
         ctx : Context
-            The context containing the HTTP session used to interact with the API.
-        parent : Active
-            Parent resource for maintaining hierarchical relationships
+            The context object containing the session and URL for API interactions
+        path : str
+            The HTTP path component for the collection endpoint
+        pathinfo : str
+            The HTTP part of the path directed at a specific resource, by default "jobs"
         uid : str, optional
-            The default field name used to uniquely identify records, by default "key"
+            The field name used to uniquely identify records, by default "guid"
         """
-        super().__init__(ctx, parent, uid)
-        self._parent = parent
+        super().__init__(ctx, path, pathinfo, uid)
 
-    @property
-    def _endpoint(self) -> str:
-        return self._ctx.url + f"v1/content/{self._parent['guid']}/jobs"
+    def _create_instance(self, path: str, pathinfo: str, **kwargs: Any) -> Job:
```

Comment:
Add in arg separator

```suggestion
    def _create_instance(self, path: str, pathinfo: str, /, **kwargs: Any) -> Job:
```